### ISSUE TITLE
Error in test_web_accessibility 

### ISSUE THREAD

**Issue by willdavidson05** [software-gardening/almanack#58]
### Existing issue check

- [X] I found no existing [issues](https://github.com/software-gardening/almanack/issues) covering this topic.

### Description

I encountered an error while running the tests. The specific failure occurred in the test_web_accessibility function. The relevant part of the error message is:
```python

    def check_subproc_run_for_nonzero(completed_proc: subprocess.CompletedProcess) -> None:
        """
        Checks subprocess.CompletedProcess for errors and displays stdout
        in a legible way through pytest.
        """
    
        try:
            # check that the build returns 0 (nothing failed)
>           assert completed_proc.returncode == 0
E           assert 1 == 0
E            +  where 1 = CompletedProcess(args=['npx', 'pa11y', PosixPath('/tmp/pytest-of-willdavidson/pytest-31/test_web_accessibility0/jupyte...ernal/modules/cjs/loader.js:1114:14)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1143:10)\n").returncode

tests/conftest.py:21: AssertionError

The above exception was the direct cause of the following exception:

build_jupyter_book = PosixPath('/tmp/pytest-of-willdavidson/pytest-31/test_web_accessibility0/jupyter-book-test-target')

    def test_web_accessibility(build_jupyter_book: pathlib.Path) -> None:
        """
        Test web accessibility for the Jupyter Book build (html pages) using
        pa11y npm package.
    
        Note: we use the command line interface for pa11y to leverage
        cross-language capability through python (pa11y is a node.js package)
        """
    
        # gather all html pages from the build
        for html_page in pathlib.Path(f"{build_jupyter_book}/_build/html").glob("*.html"):
            # check each subproc result
>           check_subproc_run_for_nonzero(
                completed_proc=subprocess.run(
                    [
                        "npx",
                        "pa11y",
                        html_page,
                    ],
                    stdout=subprocess.PIPE,
                    stderr=subprocess.PIPE,
                )
            )

tests/test_build.py:53: 

``` 

### Reproducible example

1.) Clone the repo 
2.) Execute the command: Poetry run pytest

### Versions

- Linux

### Would you like to work on a solution for this?

- [X] Yes I am willing to submit a PR!
